package.json

```diff
@@ -23,15 +23,8 @@
   "files": [
     "dist/**"
   ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/tinylibs/tinyspy.git"
-  },
+  "repository": "tinylibs/tinybench",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/tinylibs/tinyspy/issues"
```

Comment:
This isn't inferred from `repository`, but I don't think we really need it since there's already a link to the repository.